prefs.js

```diff
@@ -59,6 +59,33 @@ export default class TogglerPreferences extends ExtensionPreferences {
     rowShortcut.add_suffix(entryShortcut);
     rowShortcut.activatable_widget = entryShortcut;
 
+    // Workspaces
+    const rowWorkspaces = new Adw.ActionRow({
+      title: "Workspaces Mode",
+      subtitle: "What to do when terminal window is in different workspace"
+    })
+    group.add(rowWorkspaces)
+
+    const workspaceModes = new Gtk.StringList()
+    workspaceModes.append("Switch to opened")
+    workspaceModes.append("Move into space")
+    workspaceModes.append("New for workspace")
+
+    const entryWorkspaces = new Gtk.DropDown({
+      valign: Gtk.Align.CENTER,
+      model: workspaceModes
```

Comment:
I believe we will need to append `selected: settings.get_string("workspaces-mode"),`

Reply:
Well, it works fine with binded model at least for me:
1. I changed setting
2. Closed window of preferences and closed extensions
3. Opened extensions and preferences once again 
4. The value was set to the one I set in step 1.